layouts/partials/sidenav/sidenav-m.html

```diff
@@ -0,0 +1,43 @@
+{{ $dot := . }}
+
+<div class="mobilequicknav row align-items-center no-gutters">
+    <a href="https://www.datadoghq.com" class="col text-center">
+        {{ partial "img.html" (dict "root" $dot "src" "icons/nav_home.png" "class" "static" "alt" "home" "width" "21") }}
+        {{ i18n "home" }}
+    </a>
+    <a href="{{ "" | absLangURL }}" class="col text-center">
+        {{ partial "img.html" (dict "root" $dot "src" "icons/nav_docs.png" "class" "static" "alt" "docs" "width" "21") }}
+        {{ i18n "docs" }}
+    </a>
+    <a href="{{ "api/" | absLangURL }}" class="col text-center">
+        {{ partial "img.html" (dict "root" $dot "src" "icons/nav_mobile_api.png" "class" "static" "alt" "api" "width" "21") }}
+        {{ i18n "api" }}
+    </a>
+</div>
+
+<aside class="sidenav">
+    <div class="row sticky">
+        <div class="col">
+            <a href="{{ "" | absLangURL }}"><h4>{{ i18n "datadog_docs" }}</h4></a>
```

Comment:
yea its to reference the homepage but in the language you are on so if you are english it will go to `https://docs.datadoghq.com/` but if you are on french it will go to `https://docs.datadoghq.com/fr/`